connection/src/thick.rs

```diff
@@ -304,23 +318,8 @@ impl<CP: CredentialsProvider> BlockchainConnection for ThickClient<CP> {
         request.set_limit(limit);
 
         self.authenticated_attested_call(|this, call_option| {
-            let (header, message, trailer) = this
-                .blockchain_api_client
-                .get_blocks_full(&request, call_option)?;
-
-            // Update cookies from server-sent metadata
-            if let Err(e) = this
-                .cookies
-                .update_from_server_metadata(header.as_ref(), trailer.as_ref())
-            {
-                log::warn!(
-                    this.logger,
-                    "Could not update cookies from gRPC metadata: {}",
-                    e
-                )
-            }
-
-            Ok(message)
+            this.blockchain_api_client
+                .get_blocks_async_opt(&request, call_option)
```

Comment:
Is changing this to the async opt going to break how blocks are currently syncing for existing clients? Like should we test how the various clients sync (I'm unsure whether test coverage covers this)

Reply:
The `_full` suffix was our proposal, implemented only in our fork. The functionality has been implemented in the published crate with the `_async_opt` suffix.